sdk/python/foundation-models/system/inference/visual-question-answering/visual-question-answering-batch-endpoint.ipynb

```diff
@@ -0,0 +1,501 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Visual Question Answering Inference using Batch Endpoints\n",
+    "\n",
+    "This sample shows how to deploy `visual-question-answering` type models to a batch endpoint for inference.\n",
+    "\n",
+    "### Task\n",
+    "`visual-question-answering` takes in images and question text prompt. For each image, it answers the user's question.\n",
+    "\n",
+    "### Model\n",
+    "Models that can perform the `visual-question-answering` task are tagged with `visual-question-answering`. We will use the `Salesforce/blip-vqa-base` model in this notebook. If you opened this notebook from a specific model card, remember to replace the specific model name. If you don't find a model that suits your scenario or domain, you can discover and [import models from HuggingFace hub](../../import/import_model_into_registry.ipynb) and then use them for inference.\n",
```

Comment:
```suggestion
    "Models that can perform the `visual-question-answering` task are tagged with `visual-question-answering`. We will use the `Salesforce/blip-vqa-base` model in this notebook. If you opened this notebook from a specific model card, remember to replace the specific model name. \n",
```

Reply:
Better to add re-direction to import model when the component supports it. Currently, few modifications are required for this to work.

Reply:
fixed